nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -529,46 +549,58 @@ def _aggregate_impl(self):
         return self._container.shape
 
 
-class MinAggregator(TensorAggregatorBase):
-    def _register_reduced_input_impl(self, x: TensorType) -> None:
-        if not self._container:
-            self._container = x
+class OnlineOfflineAggregatorBase(TensorAggregatorBase, ABC):
```

Comment:
I'll rename it, this is base aggregator for both online and offline aggregators

Reply:
IMO, introducing a `base aggregator for both online and offline aggregators` is not a good idea at all since we already have `Online..`, `Offline..`, and `..Base` aggregators. It's hard to understand which one of them you need while introducing the new custom aggregator.

Reply:
I've renamed them, now there are `AggregatorBase` and `TensroAggregatorBase`, please check

Reply:
@KodiaqQ, I'll add docstring to make it clearer

Reply:
Done, please check